web-client/public/js/update-app.js

```diff
@@ -486,6 +485,14 @@ export const updateApp = grnState => {
         updaters.removeNodeColoring();
     }
 
+    if (grnState.network !== null &&  grnState.network.sheetType === "weighted") {
+        $(EXPORT_TO_WEIGHTED_SIF).parent().removeClass("startDisabled").removeClass("disabled");
+        $(EXPORT_TO_WEIGHTED_GML).parent().removeClass("startDisabled").removeClass("disabled");
+    } else if (grnState.network !== null &&  grnState.network.sheetType === "unweighted") {
+        $(EXPORT_TO_WEIGHTED_SIF).parent().removeClass("startDisabled").addClass("disabled");
+        $(EXPORT_TO_WEIGHTED_GML).parent().removeClass("startDisabled").addClass("disabled");
+    }
```

Comment:
This block doesn't do anything when `grnState.network === null`.....seems like we should accommodate that case?